x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/follower_indices_table/follower_indices_table.js

```diff
@@ -179,24 +179,28 @@ export const FollowerIndicesTable = injectI18n(
           );
         }
       }, {
-        field: 'shards',
+        field: 'isPaused',
         name: intl.formatMessage({
           id: 'xpack.crossClusterReplication.followerIndexList.table.statusColumnTitle',
           defaultMessage: 'Status',
         }),
         truncateText: true,
         sortable: true,
-        render: (shards) => {
-          return shards && shards.length ? (
-            <FormattedMessage
-              id="xpack.crossClusterReplication.followerIndexList.table.activeStatus"
-              defaultMessage="Active"
-            />
+        render: (isPaused) => {
+          return isPaused ? (
+            <EuiHealth color="warning">
```

Comment:
What do you think of using gray instead? I used gray to indicate "Inactive" status of rollup jobs because it seemed the most inert. I usually would expect yellow to indicate a non-critical problem.

![image](https://user-images.githubusercontent.com/1238659/51696924-860f3600-1fbb-11e9-9a23-1240794052fc.png)

Reply:
let's bring this up in design review - I think gray works for "stopped" in rollup jobs, but yellow reflects "paused" better to me (though the difference between "stopped" and "paused" is negligible..)